forecasters/animalia/R/featurizers.R

```diff
@@ -32,30 +24,4 @@ make_state_7dav_featurizer <- function() {
                     ~ zoo::na.locf(.x, na.rm = FALSE))) %>%
       ungroup()
   }
-}
-
-#' @rdname featurizers
-#' @export
-make_county_7dav_featurizer <- function(response_data_source = "jhu-csse",
-                                        response_signal = "confirmed_incidence_num",
-                                        n_locations = 200) {
-  response_name = rlang::sym(
-    paste0("value+0:", response_data_source, "_", response_signal))
-  function(df) {
-    locs <- df %>% 
-      group_by(.data$geo_value) %>%
-      summarise(tot = sum(!!response_name, na.rm = TRUE)) %>%
-      slice_max(order_by = .data$tot, n = n_locations) %>%
-      select(.data$geo_value) %>%
-      pull()
-    df <- df %>% 
-      filter(.data$geo_value %in% locs) %>%
-      group_by(.data$geo_value) %>% # maybe easier with modeltools::slide_by_geo
-      arrange(.data$time_value) %>%
-      mutate(across(starts_with("value"), 
-                    ~ RcppRoll::roll_meanr(.x, n = 7, na.rm = TRUE) * 7),
-             across(starts_with("value"),
-                    ~ zoo::na.locf(.x, na.rm = FALSE))) %>%
-      ungroup()
-  }
-}
+}
```

Comment:
Add an empty line here (I don't know why this is desirable, but it is.)

Reply:
Fixed.